rust/src/measurements/make_private_expr/expr_noise/test.rs

```diff
@@ -134,3 +135,55 @@ fn test_make_laplace_grouped() -> Fallible<()> {
     );
     Ok(())
 }
+
+fn check_autocalibration(
+    margin: Margin,
+    bounds: (u32, u32),
+    d_in: (u32, u32, u32),
+) -> Fallible<()> {
+    let lf_domain =
+        LazyFrameDomain::new(vec![SeriesDomain::new("A", AtomDomain::<i32>::default())])?
+            .with_margin::<&str>(&[], margin)?;
+    let expr_domain = lf_domain.select();
+
+    // Get resulting sum (expression result)
+    let m_sum = col("A")
+        .clip(lit(bounds.0), lit(bounds.1))
+        .sum()
+        .dp()
+        .noise(None, None)
+        .make_private(
+            expr_domain,
+            PartitionDistance(InsertDeleteDistance),
+            MaxDivergence::default(),
+            Some(1.),
+        )?;
+
+    let epsilon = m_sum.map(&d_in)?;
+    // autocalibration chooses a noise scale based on the sensitivity.
+    // because of this, epsilon will always work out to 1.
+    println!("epsilon: {:?}", epsilon);
```

Comment:
Curious about this: Not doing any harm, except maybe for noise on stdout, but if there were a problem, wouldn't the `assert_eq!` tell us everything we needed?

Reply:
Yes, it's not necessary.